app/forms/link_case/confirmation_form.rb

```diff
@@ -0,0 +1,28 @@
+module LinkCase
+  class ConfirmationForm < BaseForm
+    form_for LinkedApplication
+
+    attr_accessor :link_type_code
+
+    validates :link_type_code, presence: true, unless: proc { draft? }
```

Comment:
nit, this does the same thing as `proc { draft? }`

```suggestion
    validates :link_type_code, presence: true, unless: :draft?
```